downstream/modules/platform/proc-gw-organizations-exec-env.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-gw-organizations-exec-env_{context}"]
+
+= Working with execution environments
+
+When automation controller is enabled on the platform, you can review any execution environments you have set up and manage their settings within the organization resource.
```

Comment:
couple of missed attributes here-- {ControllerName} for automation controller and {ExecEnvShort} (though the attribute is singular, not plural) for execution environment.